homeassistant/components/open_router/config_flow.py

```diff
@@ -181,11 +210,22 @@ async def async_step_user(
         except Exception:
             _LOGGER.exception("Unexpected exception")
             return self.async_abort(reason="unknown")
-        options = [
-            SelectOptionDict(value=model.id, label=model.name)
-            for model in self.models.values()
-            if SupportedParameter.STRUCTURED_OUTPUTS in model.supported_parameters
-        ]
+        # Include all models for AI tasks, but prioritize structured output models
```

Comment:
Structured output is required for AI Task generate_data to be supported. It is not optional.